css/style.css

```diff
@@ -0,0 +1,487 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-family: 'lato', sans-serif;
+  list-style: none;
+}
+
+.header-one,
+.header-two,
+.mobile-menu {
+  display: none;
+}
+
+.mobile-menu {
+  background-color: #272a31;
+  position: absolute;
+  top: 0;
+  height: 50vh;
+  padding: 1rem;
+  width: 100vw;
+}
+
```

Comment:
- [ ] For a better user experience, Kindly make sure your mobile menu is styled correctly to meet the ux standards as you may notice how it's not filling the total height once the menu is activated 

### Your design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/23e207f6-043a-496c-b050-5f0b44931d56)

HInt: To fix that you may consider setting the height to `100vh`  instead of `50vh` and to avoid when you scroll the page it looks messy again you may consider setting the nav_bar container to `fix`